scripts/build-crds.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+
+cd $(dirname $0)/../
+
+# This will return non-zero until all of our objects in ./pkg/apis can generate valid crds.
+# allowDangerousTypes is needed for struct that use floats
+controller-gen crd:generateEmbeddedObjectMeta=true,allowDangerousTypes=false paths=./pkg/apis/... output:crd:dir=./pkg/crds/yamls/generated
```

Comment:
After looking at our readme we don't really have this kind of information in there, so I think dapper changes should suffice since they will be able to run with make build-crds

Reply:
I commented elsewhere, but now I see this.  Maybe something in our Makefile that can check for/install our desired version of the tool?  I can imagine a world where folks wind-up baffled when their locally generated output don't match what's coming out of dapper.